impl/src/planning.rs

```diff
@@ -624,12 +652,21 @@ impl<'planner> CrateSubplanner<'planner> {
           )
         }
       }
+
+      // Check if the dependency has been renamed
+      dep.rename.as_ref().map(|rename| {
+        renamed_dep_names.insert(
+          dep.name.clone(),
+          format!("{}", rename),
```

Comment:
can this format be shortened to rename.clone()?

Reply:
It was giving me compiler errors before for some reason, but now that we switched to the `if let` syntax I'll change it.